### ISSUE TITLE
Jellifin is not found (in reasonable time) by DLNA clients (on Android)

### ISSUE THREAD

**Issue by kernal64** [jellyfin/jellyfin#5706]
**Describe the bug**
I'm running Jellifin on a Debian (Buster) host from the "jellyfin/jellyfin" Docker-container with network mode "host". I realized, that Jellifin is not found in reasonable time by DLNA clients (on Android) like VLC, Hi Fi cast or Bubble UPnP. From checking with tcpdump within the container, I can see, that the container is receiving the ssdp:discover requests but Jellyfin is not answereing the requests. So the clients have to wait for an active announcement.

**System (please complete the following information):**
 - OS: Debian GNU/Linux 10 (buster)
 - Virtualization: Docker version 19.03.12, build 48a66213fe
 - Clients: Android
 - DLNA Clients: Hi-Fi Cast, VLC, BubbleUPnP
 - Jellyfin Version: 10.7.1
 - Playback: not sure (Transcoding w/ VAAPI is working)
 - Installed Plugins: AudioDB10.7.1.0, MusicBrainz10.7.1.0, OMDb10.7.1.0, TMDb Trailers5.2021.102.353
 - Reverse Proxy: none
 - Base URL: none
 - Networking: Host
 - Storage: local

**To Reproduce**
1. Go to VLC on Android
2. Click on "Files" (should be Files. It's named "Dateien" on German version)
3. Jellifin does not appear in the "Local Network" section. (My W-LAN Streaming Adapter MD86672 from Medion does)

or

1. Goto BubbleUPnP on Android
2. Click "Mediathek" (German, the button right to "Playlist")
3. From the drop down choose "Mediathek auswählen..." (Not "local and cloud", the other option)
4. Jellifin does not appear in the available options. (My "Medion"-DLNA Tuner does)

or

1. Goto Hi-Fi Cast on Android
2. Select Music Library
3. Select Music Source
4. Select UPNP/DLNA
5. Jellifin does not appear in the available options. (My "Medion"-DLNA Tuner does)

If you are in luck and wait long enough (minutes), Jellyfin may be suddenly there. If it's there, everthing else is working as expected, as far as I have tested it.

**Expected behavior**
I expect that Jellyfin appears within seconds as a DLNA source.

**Logs**
I found no errors in the log

**Comment by BaronGreenback** [comment]
This has been fixed in a forthcoming release.

**Comment by kernal64** [comment]
Nice to know. At first I wanted to use my mobile as a DLNA _Digital Media Controller_ to cast the music from my _Digital Media Server (Jellyfin)_ to my _Digital Media Players_. I have used _Hi-Fi Cast_ before and tested it first and came around this issue. Afterwards I cross-checked with _VLC_ and _Bubble UPnP_ and the latter looked much better than _Hi-Fi Cast_. I decided to go with _Bubble UPnP_ but as it is capable of playing Video as well, I wanted it too <greedy_smile>. I played around with it and found out, that playing Videos via DLNA is only working with my AVIs and not with my MP4s. A file in the _transcoded_-folder is generated, but the "spinning circle" spins "forever". After thinking about my options, I just installed the _Jellifin Client_ and all is working smoothly, but I think the media is routed through the mobile, right? No nned to answer, I'll just test it, starting a stream an switching off the mobile afterwards.

Cool project anyways. I'd like to spend some money. Is there a preferred method to do so?
